gui_top_bar_alpha.lua

```diff
@@ -1531,42 +1716,75 @@ local function updateAllyTeamOverflowing()
 				end
 			end
 
-
-			-- needed for exact calculations
-			local realMetalPull = metalPull
-			local realEnergyPull = energyPull
-			if BP[6] then
-				totalStallingM = BP[6]

```

Comment:
this is important to fix the low prio m and e pull bug!
It is in the wrong function, though